scripts/bootstrap.sh

```diff
@@ -20,5 +25,15 @@ fi
 # await redis availability
 /scripts/await-service.sh $REDIS_HOST $REDIS_PORT $SERVICE_READINESS_TIMEOUT
 
-# run the service
-exec /scripts/run-service.sh
+if [[ $APP_COMPONENT == "api" ]]; then
+  exec /scripts/run-api.sh
```

Comment:
changed to "run-api.sh" as the "service" represents the entire repo (including all subapps)

this pr splits all other apps out of the api, so it is the appropriate point to change this.